src/location.js

```diff
@@ -0,0 +1,10 @@
+
+navigator.geolocation.getCurrentPosition(function (position){
+    let coordonnees = []
+    coordonnees.push(position.coords.latitude)
+    coordonnees.push(position.coords.longitude)
+    return coordonnees
+});
```

Comment:
Why did you made this function to export if you never use it on the other files ?

Reply:
And I am not sure it works

Reply:
I have delete this file